index.js

```diff
@@ -1,24 +1,79 @@
-function rot13(str) { // LBH QVQ VG!
-var shifted = 13;
-var splitted = str.split('');
-var arr = [];
-var inCharCode = splitted.forEach(function(element) {
-arr.push(element.charCodeAt(0));
-return arr;
-});
-var shifterArray = arr.map(function(element) {
-return element += shifted;
-})
-var someString = shifterArray.join( );
-var res = String.fromCharCode(someString);
-var resultArray = shifterArray.map(function (x) {
-return parseInt(x, 10);
-});
-var symbols = resultArray.forEach(function(val) {
-return String.fromCharCode(val);
-})
-return symbols;
+/**
+ * Used for tests, throws an error if value is not equal to expected value
+ */
+function assertEq(value, expected) {
+    if (value != expected) {
+        throw 'Expected ' + value + ' to be ' + expected;
+    }
 }
 
-// Change the inputs below to test
-rot13("SERR PBQR PNZC");
\ No newline at end of file
+/**
+ * @param {String} str Source string
```

Comment:
Is it good practice for me to start use JSDoc, or it is not necessary at beginning?

Reply:
Documentation is always must have. Especially for beginners =)